Visual Studio 2022 support

Intrets opened the pull request:
I started with an empty VSIX project and added `TabGroupJump.cs`. Had to add a reference to `Microsoft.VisualStudio.Platform.WindowManagement`, but nothing else. Building might not work for everyone (?).

As a result some files also didn't get generated in a new project, I don't think they are needed so I left them out.

Intrets committed:
Visual Studio 2022 support

Intrets committed:
added more suggestions made by linter

Intrets committed:
tabs snuck in

Intrets committed:
actually fixed the spacing instead of messing it up further

Intrets committed:
removed visual studio 2019 versions

Intrets commented:
I just realised I really bruteforced it and it will ONLY work for 2022, not add support for 2022.

zastrowm commented:
I'm okay with only having 2022 support; uploading as different extension is one possibly, which is the path I'll most likely if needed.

I need to update from preview to 2022 and check this out locally.  But if it works out, I should be able to upload within the next week or so.

Intrets commented:
~~I don't think the things I did have made much sense. On second look it is screaming piles of warnings at me about referencing local .dlls.~~ added old `VSPackage.resx` back
~~And then there was something with a refernce to `Microsoft.VisualStudio.Platform.WindowManagement` that I solved by giving it a hint to the default VS2022 installation location, which of course is not good either.~~ should be fixed

Intrets committed:
Added VSPackage and removed a dependency

-Added back old VSPackage.resx
-Removed Microsoft.VisualStudio.Platform.WindowManagement dependency - it was only used for WindowFrame.onScreen and the IVsWindowScreen interface checks that for a WindowFrame object.

Intrets committed:
more sensible name

Intrets committed:
fixed misplaced icon file

Intrets committed:
added back old packages.config

Intrets committed:
Changed file to original line endings

Intrets commented:
On the topic of uploading as a different extension, all the plugins I use have appeared as a seperate plugin for VS2022, so that will probably be necessary.
But breaking old targets like I did is maybe a bit unnecessary.

Intrets committed:
undo capitalization error

jonas-lomholdt commented:
I would love to get this thing uploaded! It's crazy there's no native support for this in VS2022 😄 
If I can help in some way, I'm up for that.

jagt commented:
I just realized that I can just download this repo and build the extension and use it.
So here's my local build. It works on my box 😉 

https://github.com/jagt/vs-NavigateTabGroups/releases/tag/vs2022

Also thanks for all your good works! I literally can't use VS without this extension.
